arch/cortex-m7/src/mpu.rs

```diff
@@ -0,0 +1 @@
+../../cortex-m3/src/mpu.rs
```

Comment:
This should probably be copied or otherwise imported using Rust methods, not a symlink. Explains why you applied the M7 changes to the M3 files.